recipe/meta.yaml

```diff
@@ -7,51 +7,48 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://github.com/PDAL/python/archive/{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 
 build:
-  number: 2
-  skip: true  # [(win and vc<14) or (py<36) or (py<38 and linux)]
+  number: 0
+  skip: true  # [win or s390x or py<38]
 
 
 requirements:
-  run_constrained:   # [osx]
-    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}      # [osx]
   build:
     - {{ compiler('cxx') }}
-    - make
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
-    - pybind11                               # [build_platform != target_platform]
-    - curl
+    - make  # [unix]
     - cmake
-    - pdal
+    - pdal 2.5.3
     - ninja
-    - scikit-build
+    - scikit-build 0.15.0
```

Comment:
IMO, scikit-build is needed only in `host` as it's a python package. Could you check it?